modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/source/ExtensionMessageSource.java

```diff
@@ -621,9 +625,17 @@ protected void doInitialise() throws InitialisationException {
     flowConstruct = new LazyValue<>(() -> (FlowConstruct) componentLocator.find(getRootContainerLocation()).orElse(null));
     messageProcessContext = createProcessingContext();
     if (shouldRunOnThisNode()) {
+      LOGGER
+          .debug("Message source '{}' on flow '{}' is being initialised, it is running on the primary node or no cluster restrictions are being applied.",
```

Comment:
disambiguate this. It should say wether or not it's the primary node, and if it isn't it should explain why is it starting anyway